cv32e40p/tests/programs/custom/pulp_hardware_loop_interrupt_test/pulp_hardware_loop_interrupt_test.h

```diff
@@ -0,0 +1,104 @@
+// This is free and unencumbered software released into the public domain.
```

Comment:
It seems that neither OpenHW or Dolphin should claim copyright on this file, but it would be useful to add an SDPX header.

Reply:
Same